migrations/20180327173047_add_supplier_contracts.js

```diff
@@ -0,0 +1,18 @@
+var constants = require('../models/constants.js');
+exports.up = function(knex, Promise) {
+  return Promise.all([
+
+        // Suppliers table
+        knex.schema.createTable(constants.SUPPLIERS_CONTRACTS_TABLE_NAME, function(table) {
+            table.timestamps();
+            table.integer('supplier_id', 9).unsigned().primary();
+            table.string('contract_name', 100);
```

Comment:
i think this should be called path or url

Reply:
I am changing `contract_name` to `file_name`, because it's just the leaf name and not a full path.
This string is also used as a key for S3 bucket.